schema/Makefile

```diff
@@ -0,0 +1,93 @@
+# The version of the open source schema that this version of the
+# enterprise schema depends on.  Change this as new versions become
+# available.
+OSC_SCHEMA_VERSION=1.0.4
```

Comment:
@tylercloke In the other PR I updated all the upgrade logic to run sqitch directly.  I think we could probably nuke these make files completely and maybe just make a single schema target in the top level makefile if we want one for some reason.